test/flycheck-test.el

```diff
@@ -50,12 +50,13 @@
 (require 'flycheck-ert)
 
 ;; Make a best effort to make Coq Mode available
-(mapc (lambda (dir)
-        (add-to-list 'load-path (expand-file-name "coq/" dir)))
-      '("/usr/share/emacs/site-lisp/"
-        "/usr/local/share/emacs/site-lisp/"))
+;; (mapc (lambda (dir)
+;;         (add-to-list 'load-path (expand-file-name "coq/" dir)))
+;;       '("/usr/share/emacs/site-lisp/"
+;;         "/usr/local/share/emacs/site-lisp/"))
+(add-to-list 'load-path "/usr/share/emacs/site-lisp/")
 
-(autoload 'coq-mode "coq")
+(autoload 'coq-mode "gallina")
```

Comment:
Sure: in the VM, `coq-mode` was provided by `gallina.el`, not `coq.el`.  The ArchLinux package is compiled straight from [the source](https://coq.inria.fr/download), and I've just checked: the latest archive has `gallina.el` providing `gallina`; no trace of `coq.el` or `(provide 'coq)`.

So this was required in order for me to run the tests inside the VM.

But I don't use Coq, so I might have missed something.  Were you able to run the tests using 8.6?